.github/workflows/deploy_to_git.yml

```diff
@@ -154,3 +160,91 @@ jobs:
         run: gh pr merge --auto --squash --delete-branch ${{ steps.cpr.outputs.pull-request-number }}
         env:
           GITHUB_TOKEN: ${{ steps.generate-token.outputs.token }}
+
+  get-version: # emit package version using `cargo tree -i`
+    name: Get Package Version
+    runs-on: ubuntu-latest
+    needs: [build_shared_library, deploy_to_git]
+    if: |
+      always() && 
+      ( needs.build_shared_library.result=='success' || needs.build_shared_library.result=='skipped' ) &&
+      ( github.event.pusher.name == 'finschia-auto-pr[bot]' || needs.deploy_to_git.outputs.updated=='false' )
```

Comment:
Does this only run after finschia-auto-pr[bot]? It should run also after a PR that changes the version even if it has the latest build of shared libraries. This looks it does not run, isn't it?

Reply:
```
( github.event.pusher.name == 'finschia-auto-pr[bot]' || needs.deploy_to_git.outputs.updated=='false' )
```
Because this condition uses operator "or", so it will run the following section when either the pusher bot or when the shared libraries is already up to date with the latest version.
```
always() && 
      ( needs.build_shared_library.result=='success' || needs.build_shared_library.result=='skipped' ) &&
```
This part is designed to be executed even if the first task is skipped.